tests/scripts/test_config.sh

```diff
@@ -14,29 +14,29 @@ export XDG_CONFIG_HOME=/nowhere
 
 sed -e '/base_url/c\base_url = "http://badurl"' "$good_settings" > "$bad_settings"
 
-expect_succ pulp --config "$good_settings" file repository list
-expect_succ pulp --config "$bad_settings" --base-url "$PULP_BASE_URL" file repository list
+expect_succ pulp --config "$good_settings" task list
```

Comment:
I assume this section is to clean up the test so it doesn't need a plugin to test the configs, yeah?

Reply:
Yes, i found this after running against a pulp without pulp_file. Uhg...
It was either this or adding a test requirement on pulp_file. But why skip a test for a bad reason.